keyboards/kprepublic/cstc40/single_pcb/rules.mk

```diff
@@ -0,0 +1,3 @@
+# Build Options
+LTO_ENABLE = no
+TRI_LAYER_ENABLE = yes
```

Comment:
```suggestion
# This file intentionally left blank
```

Reply:
Fixed with the same comment of the previous suggestion.